### PR TITLE
Fix bugs with custom_logL_priors

### PR DESCRIPTION
**Pull request by chinyitan**

### CODE REVIEW

File: dolphin/processor/files.py [hunk 1]
```diff
@@ -290,7 +290,7 @@ def save_output_h5(self, lens_name, model_id, output):
                                             )
                     subgroup.create_dataset('param_list',
                                             data=np.array(single_output[2],
-                                                          dtype='S10')
+                                                          dtype='S25')
```
Comment (ajshajib) [inline dolphin/processor/files.py:293]: Does choosing "S25" solve the clipped parameter name issue? I thought, it was lenstronomy that was doing the clipping to begin with.

**Review by ajshajib — CHANGES_REQUESTED** [review]

File: dolphin/analysis/output.py [hunk 2]
```diff
@@ -270,8 +271,16 @@ def plot_model_overview(self, lens_name, model_id=None,
                                             v_min=-3)
         model_plot.source_plot(ax=axes[1, 0], deltaPix_source=0.02, numPix=100,
                                band_index=band_index, v_max=v_max, v_min=v_min)
-        model_plot.convergence_plot(ax=axes[1, 1], band_index=band_index,
-                                    cmap=convergence_cmap)
+        if not show_source_light:
```
[the remark this answers is not in the document]
Comment (chinyitan) [reply dolphin/analysis/output.py:274]: I think is more a my personal preference  but I found that the convergence plot usually contains very similar information as the magnification plot. However, it often useful to see the lensed source light plot to see if the lens features are recreated properly (particular for fainter arcs). Therefore having the  lensed source light information often make going through many lenses much easier for me.

File: dolphin/analysis/output.py [hunk 3]
```diff
@@ -270,8 +271,16 @@ def plot_model_overview(self, lens_name, model_id=None,
...
+            model_plot.convergence_plot(ax=axes[1, 1], band_index=band_index,
+                                        cmap=convergence_cmap)
+        else:
+            model_plot.decomposition_plot(ax=axes[1, 1],
```
[the remark this answers is not in the document]
Comment (chinyitan) [reply dolphin/analysis/output.py:278]: The lensed in source light in the model, though I did notice that you had the arc in the data in your paper.

File: dolphin/processor/files.py (hunk 1 above)
Comment (chinyitan) [reply dolphin/processor/files.py:293]: Yes, surprisingly it does. I also changed the parameter name font from 10 to 8 to accommodate the longer parameter name

**Comment by chinyitan** [comment]
Closed the PR, so we can centralize all the dino-i changes (including the changes here) at a new PR.

**Pull request closed without merging**